article/app/pages/StoryHtmlPage.scala

```diff
@@ -0,0 +1,66 @@
+package pages
+
+import common.Edition
+import conf.switches.Switches.SurveySwitch
+import html.HtmlPageHelpers._
+import html.Styles
+import model.{ApplicationContext, Page}
+import play.api.mvc.RequestHeader
+import play.twirl.api.Html
+import views.html.fragments._
+import views.html.fragments.commercial.{pageSkin, survey}
+import views.html.fragments.page._
+import views.html.fragments.page.body.{bodyTag, breakingNewsDiv, skipToMainContent, twentyFourSevenTraining}
+import views.html.fragments.page.head.stylesheets.{criticalStyles, styles}
+import views.html.fragments.page.head.{fixIEReferenceErrors, headTag, titleTag}
+
+object StoryHtmlPage {
+
+  def allStyles(implicit applicationContext: ApplicationContext): Styles = new Styles {
+    override def criticalCss: Html = criticalStyles()
+    override def linkCss: Html = stylesheetLink("stylesheets/content.css")
+    override def oldIECriticalCss: Html = stylesheetLink("stylesheets/old-ie.head.content.css")
+    override def oldIELinkCss: Html = stylesheetLink("stylesheets/old-ie.content.css")
+    override def IE9LinkCss: Html = stylesheetLink("stylesheets/ie9.head.content.css")
+    override def IE9CriticalCss: Html = stylesheetLink("stylesheets/ie9.content.css")
+  }
+
+  def html(
+    header: Html,
+    content: Html,
+    maybeHeadContent: Option[Html] = None
+  )(implicit page: Page, request: RequestHeader, applicationContext: ApplicationContext): Html = {
+
+    val head: Html = maybeHeadContent.getOrElse(Html(""))
+    val bodyClasses = defaultBodyClasses() ++ Map(
+      ("is-immersive", Page.getContent(page).exists(_.content.isImmersive))
```

Comment:
Is it fine if I don't specify the `is-immersive-interactive` class for articles?

Reply:
I can't think this should be okay, since that should be covered by applications.